pkg/manifest/os.go

```diff
@@ -311,6 +315,16 @@ func (p *OS) getBuildPackages(distro Distro) []string {
 		packages = append(packages, "openscap-utils")
 	}
 
+	if p.BootcConfig != nil {
+		switch distro {
+		case DISTRO_EL8:
+			packages = append(packages, "python3-pytoml")
+		case DISTRO_EL10:
```

Comment:
Did `DISTRO_EL10` need a different package name or can it also 'just' use the default?

Reply:
For EL10 we don't install anything because the plan is to use the stdlib `tomllib`.
That said, I'm not sure if we actually modified the stages to import it properly.

Reply:
Ack :)